connect.go

```diff
@@ -19,16 +19,18 @@ func dial(network string, hosts []string, noDelay bool, r, w time.Duration, logf
 			return v
 		}
 		conn  net.Conn
-		index = abs(int(atomic.AddInt32(&tick, 1)))
+		ident = abs(int(atomic.AddInt32(&tick, 1)))
```

Comment:
It'd be nice not to make tick global, or to not force it's use. I'm happy as a caller to provide a list of hosts sorted by my preference, but the library will not actually use them in the order because of the advancing ticker.